genai-perf/genai_perf/wrapper.py

```diff
@@ -66,7 +66,7 @@ def add_inference_load_args(args: Namespace) -> List[str]:
     def add_payload_args(args: Namespace) -> List[str]:
         cmd = []
         if args.prompt_source == PromptSource.PAYLOAD:
-            cmd += ["--fixed-schedule"]
+            cmd += ["--session-concurrency=2"]
```

Comment:
A hard-coding to allow GAP to run session concurrency mode until [lkomali-support-session-args-synthetic-multiturn](https://github.com/triton-inference-server/perf_analyzer/compare/main...lkomali-support-session-args-synthetic-multiturn) gets merged.